frontend/src/metabase/search/components/SearchFilterDatePicker/SearchFilterDatePicker.tsx

```diff
@@ -2,9 +2,9 @@ import { useState } from "react";
 import { t } from "ttag";
 import { SearchFilterApplyButton } from "metabase/search/components/SearchFilterPopoverWrapper/SearchFilterPopoverWrapper";
 import { filterToUrlEncoded } from "metabase/parameters/utils/date-formatting";
-import DatePicker from "metabase/query_builder/components/filters/pickers/DatePicker/DatePicker";
-import type { DateShortcutOptions } from "metabase/query_builder/components/filters/pickers/DatePicker/DatePickerShortcutOptions";
-import { DATE_SHORTCUT_OPTIONS } from "metabase/query_builder/components/filters/pickers/DatePicker/DatePickerShortcutOptions";
+import DatePicker from "metabase/admin/datamodel/components/filters/pickers/DatePicker/DatePicker";
```

Comment:
It'd be nice to add named exports here.

```suggestion
import DatePicker from "metabase/admin/datamodel/components/filters/pickers/DatePicker";
```